jekyll/_cci2/release/configure-your-kubernetes-components.adoc

```diff
@@ -63,11 +63,16 @@ spec:
   template:
     metadata:
       labels:
-        app: example-deployment
-        version: 1.0.0
+        app: example-deployment      
+        circleci.com/component-name: example-deployment
+        circleci.com/version: 1.0.0
 ----
 
-NOTE: Do not add `version` to the selector labels. These labels can not be changed later, and `version` changes frequently.
+NOTE: Do not add `circleci.com/version` and `circleci.com/component-name` to the selector labels. These labels can not be changed later, and `circleci.com//version` should change for every new release.
+In the provided example `app` is being used in the selector, but any other label can be used in its place.
+WARNING: if you were using the release agent prior to version `1.2.0` you were using `app` and `version` in place of `circleci.com/component-name` and `circleci.com/version`. While `app` and `version` are still supported, we recommend to migrate to the new labels as soon as possible.
+Support for the old labels will be dropped in one of the next releases. 
+Also be aware that, after migrating to the new labels, rolling back to versions that used the old labels will be supported only for deployments and rollouts managed through Helm.
```

Comment:
```suggestion

[WARNING]
===
If you were using the release agent prior to version `1.2.0` you were using `app` and `version` in place of `circleci.com/component-name` and `circleci.com/version`. While `app` and `version` are still supported, we recommend to migrate to the new labels as soon as possible.

Support for the old labels will be dropped in a future release.
 
After migrating to the new labels, rolling back to versions that used the old labels will be supported only for deployments and rollouts managed through Helm.
```

Reply:
Maybe a warning like this should be added to the update guide too? https://circleci.com/docs/release/update-the-kubernetes-release-agent/

Reply:
Yes it makes sense. I will add it

Reply:
I added it.